### PR TITLE
Add Vim- and Emacs-like up/down keybindings

### PR DESCRIPTION
**Pull request by dylnclrk**
## Motivation

When I tried `bundle ui` for the first time, I found myself trying to move around the list with n/p and j/k. `yarn upgrade-interactive` listens for j/k keypresses to move around the table. Similarly, many CLIs support n/p to move up and down.

## Proposed Change

Add listeners for those keys as described in the [tty-prompt docs](https://github.com/dylnclrk/bundle_update_interactive/pull/new/add-vim-emacs-keybindings). To be honest, I wasn't totally sure what was going on in this file, so I'm not sure if this is the best place to set up these listeners.

Screen Recording 2024-07-14 at 3 10 17 PM

### CODE REVIEW

**Review by mattbrictson — CHANGES_REQUESTED** [review]
Looks good! RuboCop is complaining about the `if` conditionals, see my comment below.

Once CI is passing I'll test locally and merge it! 🙇

File: lib/bundle_update_interactive/cli/multi_select.rb [hunk 1]
```diff
@@ -51,6 +53,18 @@ def prompt
...
+    def add_keybindings
+      tty_prompt.on(:keypress) do |event|
+        if %w[k p].include?(event.value)
+          tty_prompt.trigger(:keyup)
+        end
+
+        if %w[j n].include?(event.value)
+          tty_prompt.trigger(:keydown)
+        end
+      end
```
Comment (mattbrictson) [inline lib/bundle_update_interactive/cli/multi_select.rb:65]: Hmm, GitHub lost my review comment...? Let me try again.

```suggestion
      tty_prompt.on(:keypress) do |event|
        tty_prompt.trigger(:keyup) if %w[k p].include?(event.value)
        tty_prompt.trigger(:keydown) if %w[j n].include?(event.value)
      end
```

Comment (dylnclrk) [reply lib/bundle_update_interactive/cli/multi_select.rb:65]: Ah--sorry I didn't check rubocop. I will fix when I'm back at my computer.

Comment (dylnclrk) [reply lib/bundle_update_interactive/cli/multi_select.rb]: OK, resolved :)

Thanks, Matt!

**Review by mattbrictson — APPROVED** [review]
Works great for me locally. Thanks for the PR! 🚀

**Comment by mattbrictson** [comment]
Released on 0.2.0 https://github.com/mattbrictson/bundle_update_interactive/releases/tag/v0.2.0
